images/utapi/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-alpine
+
+WORKDIR /usr/src/app
+
+COPY ./package.json ./yarn.lock /usr/src/app/
+
+RUN apk add git \
+            build-base \
+            openssl \
+            python && \
+    yarn install --production --frozen-lockfile && \
```

Comment:
I would suggest adding a .yarnrc file

And move the --production flag as an environment variable so that docker file can be reused.
`ARG NODE_ENV=production`

Reply:
Good suggestion, The utapi dockerfile slipped in by accident though this PR should really only be warp 10 stuff